From 7d53eefc547857e43cf772c0545f54fc18dec1e8 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 15 May 2020 12:00:58 +0300
Subject: [PATCH] [SW-11875] goya2: export arcboot functions

This patch marks following symbols of arcboot rom to be exported. These
symbols will be used by ppboot to perform verification of preboot image.

dma_wait_finish
security_i2c_s_enable
secureboot_verify_image
dma_copy_image
spi_flash_readl

Change-Id: I0e3b2729ebc55c76fd7b8c0be197490446482c7b
---

diff --git a/app/arcboot_rom.txt b/app/arcboot_rom.txt
index 81056c6..92277c6 100644
--- a/app/arcboot_rom.txt
+++ b/app/arcboot_rom.txt
@@ -1 +1,6 @@
 # put list of symbols, one per line, to be exported
+dma_wait_finish
+security_i2c_s_enable
+secureboot_verify_image
+dma_copy_image
+spi_flash_readl
